tests/execution/test-grpc-nested-optional.md

```diff
@@ -1,5 +1,5 @@
 ---
-error: true
+error: false
```

Comment:
isn't it intentional to get the validation error here?

so right now validation fails if we mark fields for proto as required and we need to have a test for this.